model/system.go

```diff
@@ -155,3 +158,14 @@ func SendWarnMetricAckFromJson(r io.Reader) *SendWarnMetricAck {
 	json.NewDecoder(r).Decode(&swma)
 	return swma
 }
+
+type AWSMeterReport struct {
```

Comment:
Looks like this model is not relevant to any other part of the application, should we move it into the service itself? And looks like is not serialized anywhere, so maybe we can remove the json annotations and the `ToJSON`.

Reply:
I prefer to keep the ToJSON, I find it useful in debugging and I don't have to add and then remove it.

Reply:
Sounds good to me